site/content/blog/2022-07-01-whats-new-in-svelte-july-2022.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "What's new in Svelte: July 2022"
+description: "Faster SSR, language tools improvements and a new paid contributor!"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+From faster SSR to support for Vitest and Storybook in SvelteKit, there's a lot to cover in this month's newsletter...
+
+So let's dive in!
+
+## OpenCollective funding drives Svelte forward
+
+Svelte supporters have donated approximately $80,000 to [the project on OpenCollective](https://opencollective.com/svelte). We're happy to share that the funds are being drawn on to move Svelte forward in a meaningful way. **[@gtm-nayan](https://github.com/gtm-nayan)** has begun triaging and fixing SvelteKit issues this past month as a paid contributor to the project to help us get SvelteKit to a 1.0 level of stability! @gtm-nayan has been an active member of the Svelte community for quite some time and is well known for writing the bot that helps keep our Discord server running. We're happy that this funding has allowed Svelte to get much more of his time.
```

Comment:
`@gtm-nayan` not sure if it will render correctly in page if there is `@` without any "fences".

Reply:
LGTM:
<img width="929" alt="image" src="https://user-images.githubusercontent.com/7872348/176940984-18715f49-3df9-46e3-89e5-f99d798b1016.png">